ci/deploy-exclude.txt

```diff
@@ -0,0 +1,8 @@
+# Add files or patterns to exclude from the built branch here.
```

Comment:
Should we name this `.deployignore` for consistency with `gitignore` and `svnignore`?

Reply:
Sold. I've renamed it.